modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm

```diff
@@ -634,7 +662,7 @@ GLOBAL_LIST_INIT(loadout_miscunders, generate_loadout_items(/datum/loadout_item/
 
 /datum/loadout_item/under/formal/formaldressred
 	name = "Formal Red Dress"
-	item_path = /obj/item/clothing/under/misc/formaldressred
+	item_path = /obj/item/clothing/under/dress/skyrat/redformal
```

Comment:
There's a lot of typepaths in here, but please ensure you keep them following the `snake_case` convention.

I know it's our bad for not being thorough enough enforcing it, but yeah. For consistency reasons, I'd prefer if we made efforts to stick with it a bit more.
```suggestion
	item_path = /obj/item/clothing/under/dress/skyrat/red_formal
```

Reply:
same as last, I entirely forgot because the originals didn't use snakecase. It'll be fixed